src/Chapter1.hs

```diff
@@ -653,7 +662,10 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+    | n `div` 10 /= 0 = firstDigit (n `div` 10)
+    | otherwise = n
```

Comment:
Great!
I see, that for a negative number like -19, this implementation will return the negative number.